bin/installingdependancies.js

```diff
@@ -27,42 +27,65 @@ const installingDependancies = (details) => {
         databaseAdapter = "pg"
     }
 
-    // For now, shell command is run inside this file, but need to be removed later
-    const cmd = `
-        cd ${appBaseDirectory}
-        set -e 
-        echo setting up git for you application
-        git init
-        echo Will install your dependancies shortly....
-        echo We are installing dependancies, please wait...
-        npm install ${framework.toLowerCase()} 
-        npm install body-parser 
-        npm install cors 
-       ${ orm && orm !== 'No ORM' ? `npm install ${orm.toLowerCase()}` : ''}
-        ${databaseAdapter ? `npm install ${databaseAdapter}` : ''}
-        npm install dotenv
-        npm install bcrypt
-        echo Install dev dependancies, please wait...
-        npm install -D @babel/cli 
-        npm install -D @babel/core 
-        npm install -D @babel/node 
-        npm install -D @babel/preset-env 
-        npm install -D chai-http 
-        ${expectationLibrary ? `npm install -D ${expectationLibrary.toLowerCase()}` : ''}
-        ${testFramework ? `npm install -D ${testFramework.toLowerCase()}` : ''} 
-        npm install -D nyc
-        echo adding files to git and committing
-        git add . && git commit -m "create app and set up app"
-    `
-
-    // Install dependancies and dev dependancies
-    exec(cmd, (error, stdout, stderror) => {
-        if (error) {
-            chooseConsoleColorText(colorSet.error, `exec error: ${exec}`);
-        }
-        chooseConsoleColorText(colorSet.normal, `stdout: ${stdout}`);
-        chooseConsoleColorText(colorSet.error, `stderror: ${stderror}`);
-        chooseConsoleColorText(colorSet.log, '------------Thanks for being patient-------------');
+    /**
+     * let commands be in an array 
+     * then, loop through each array 
+     * install each dependancy one by one
+     */
+    let cdBaseDirectory = `cd ${appBaseDirectory}`
+    const commands = [
+        `${cdBaseDirectory}`,
+        `set -e`,
+        `echo Will install your dependancies shortly....`,
+        `echo We are installing dependancies, please wait...`,
+        `${cdBaseDirectory} && npm install ${framework.toLowerCase()}`,
```

Comment:
Tried having a temp `package.json` file with the deps defined there and just run npm install?

Reply:
What about how to manage versions of the same dependencies?

Reply:
Or any example file with the deps will be highly appreciated.

Reply:
> https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file 
I am checking on this line though still analyzing the best way